axelrod/strategies/titfortat.py

```diff
@@ -636,3 +636,31 @@ def strategy(self, opponent: Player) -> Action:
 
         # Otherwise play previous move
         return self.history[-1]
+
+@FinalTransformer((D, ) name_prefix=None)
+class Alexei(Player):
+    """
+    Plays similar to Tit-for-Tat, but always defect on last turn.
+    
+    Names:
+
+    - From http://lesswrong.com/lw/7f2/prisoners_dilemma_tournament_results/    
+    """
+
+    name = 'Alexei'
+    classifier = {
+        'memory_depth': 1,
```

Comment:
This strategy has memory depth: `float('inf')` as it needs to keep count of how many turns it has played to be able to know when to defect on the last round. 

Fixing this will fix the other test failure as it was being picked up as a basic strategy (which it is not).